matminer/featurizers/composition.py

```diff
@@ -254,6 +255,83 @@ def citations(self):
     def implementors(self):
         return ["Jiming Chen", "Logan Ward", "Anubhav Jain", "Alex Dunn"]
 
+class Meredig(BaseFeaturizer):
+    """
+    Class to calculate features as defined in Meredig et. al.
+    """
+
+    def __init__(self):
+        self.data_source = MeredigData()
+
+        self.features = [e.strip()+" fraction" for e in self.data_source.element_names]
+        self.features = self.features+["mean AtomicWeight", "mean Column", "mean Row", "range AtomicNumber", "mean AtomicNumber",
+        "range AtomicRadius", "mean AtomicRadius", "range Electronegativity", "mean Electronegativity", "mean NsValence",
+        "mean NpValence", "mean NdValence", "mean NfValence", "fraction NsValence",
+        "fraction NpValence", "fraction NdValence", "fraction NfValence"]
+        # Initialize stats computer
+        self.pstats = PropertyStats()
+
+    def featurize(self, comp):
+        """
+        Get elemental property attributes
+
+        Args:
+            comp: Pymatgen composition object
+
+        Returns:
+            all_attributes: Specified property statistics of features
+        """
+
+        #First 103 features are element fractions
+        n_elements = 103
```

Comment:
You should make a call out to the ElementFraction featurizer rather than implementing your own version here.

Reply:
Changed